templates/auth/auth.go

```diff
@@ -0,0 +1,37 @@
+package auth
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+)
+
+func getUserInfoFromHeaders(r *http.Request) (error, string, string) {
```

Comment:
How about returning a user info struct rather than strings?

Reply:
Oh this is just private. Maybe think about how to expose this so other routes can check it too?